diff -Nurb pdf2html-1.4/Makefile pdf2html-1.4.new/Makefile
--- pdf2html-1.4/Makefile	2003-09-16 13:12:55.000000000 +0200
+++ pdf2html-1.4.new/Makefile	2004-12-28 04:15:34.917323360 +0100
@@ -1,19 +1,21 @@
-CFLAGS= -O6 -Wall -fomit-frame-pointer -funroll-loops -fstrength-reduce -malign-functions=4
-LDFLAGS= -s
+CC ?= cc
+CFLAGS ?= -O2
+override CFLAGS += -Wall -fomit-frame-pointer -funroll-loops -fstrength-reduce -falign-functions=4
+LDFLAGS ?= -s
 
 all: pbm2png pbm2eps9
 
 pbm2png: pbm2png.c
-	gcc $(CFLAGS) $(LDFLAGS) -lpng pbm2png.c -o pbm2png
+	$(CC) $(CFLAGS) $(LDFLAGS) -lpng pbm2png.c -o pbm2png
 
 pbm2eps9: pbm2eps9.o printer.o
-	gcc $(LDFLAGS) pbm2eps9.o printer.o -o pbm2eps9
+	$(CC) $(LDFLAGS) pbm2eps9.o printer.o -o pbm2eps9
 	
 pbm2eps9.o: pbm2eps9.c printer.c printer.h
-	gcc -c $(CFLAGS) pbm2eps9.c -o pbm2eps9.o
+	$(CC) -c $(CFLAGS) pbm2eps9.c -o pbm2eps9.o
 
 printer.o: printer.c printer.h
-	gcc -c $(CFLAGS) printer.c -o printer.o
+	$(CC) -c $(CFLAGS) printer.c -o printer.o
 
 clean:
 	rm -f pbm2png *.o pbm2eps9
